test/new-e2e/tests/agent-metrics-logs/log-agent/listener/tcp_test.go

```diff
@@ -62,7 +62,7 @@ func (d *dockerSuite) TestLogsReceived() {
 
 	stdout, stderr, err := d.Env().Docker.Client.ExecuteCommandStdoutStdErr("logger-app", cmd...)
 	require.NoError(d.T(), err)
-	assert.Empty(d.T(), stderr)
+
```

Comment:
Note for reviewers: I had to remove this assertion I added in the first place, the command execution returns `stderr`